<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Controle MQTT - Rasp LoRaWAN+</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>241</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Entradas analógicas</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>ADC1:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblADC1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>ADC2:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblADC2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>ADC3:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblADC3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ler_adcs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Ler</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>221</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Conexão MQTT</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Broker:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Porta:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblStatus_MQTT">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>201</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Aguardando conexão...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConectaMQTT">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>201</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Conectar / desconectar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtBroker">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>rasppi3B.local</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPorta">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>1883</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Controle de DAC</string>
    </property>
    <widget class="QDial" name="dialValor_DAC">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>50</width>
       <height>64</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QLabel" name="lblValor_dial">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0%</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblValor_DAC_Volt">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>0V</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>241</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>OLED</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Linha 1:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtOLED_Linha1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>21</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Linha 2:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtOLED_Linha2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>21</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Linha 3:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtOLED_Linha3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>21</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Linha 4:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtOLED_Linha4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>21</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Linha 5:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtOLED_Linha5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>21</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Linha 6:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtOLED_Linha6">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>21</number>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEscreve_OLED">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>221</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Escrever no OLED</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>231</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>LoRaWAN</string>
    </property>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>211</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Mensagem para enviar (uplink):</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEnvia_msg_Lorawan">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Envia mensagem</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtMsg_Uplink_Lorawan">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>211</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>30</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>221</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Mensagem recebida (downlink):</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblMsg_downlink_Lorawan">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>211</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>481</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Gostou?</string>
    </property>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>211</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Visite meu site:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnVisitar_site">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>461</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>www.pedrobertoleti.com.br</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>494</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
